### PR TITLE
422 update rest framework

### PR DESCRIPTION
**Pull request by tdowds**
Changed our expectation of the response object from the server when there are errors.

### CODE REVIEW

File: huxley/api/serializers/school.py [hunk 1]
```diff
@@ -1,22 +1,19 @@
 # Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
 # Use of this source code is governed by a BSD License (see LICENSE).
-
-from drf_compound_fields.fields import ListField
 from rest_framework import serializers
 
 from huxley.api import validators
 from huxley.api.serializers.fields import DecimalField
```
Comment (kmeht) [inline huxley/api/serializers/school.py]: We still using this?

File: huxley/settings/logging.py [hunk 2]
```diff
@@ -40,7 +40,7 @@
         'huxley.api': {
             'handlers': ['database'],
             'level': 'DEBUG',
-            'propagate': True, #Set to false to elimninate logs in the console
+            'propagate': False, #Set to false to elimninate logs in the console
```
Comment (kmeht) [inline huxley/settings/logging.py]: Is this the new default behavior?

File: requirements.txt [hunk 3]
```diff
@@ -4,8 +4,7 @@ coverage==3.7.1
 cssmin==0.2.0
 django-pipeline==1.3.24
 django-pipeline-browserify==0.1.0
-djangorestframework==2.4.3
-drf-compound-fields==0.2.2
```
Comment (kmeht) [inline requirements.txt]: Nice, love removing dependencies!

**Comment by kmeht** [comment]
Just curious, what's the deal with renaming `country_preferences` to `countrypreferences`?

Diff looks good to merge though. Please, for the love of God, squash the commits and add an informative commit description :)

**Comment by tdowds** [comment]
Closed via https://github.com/bmun/huxley/commit/1f3d87fd61291e87ff0e11829bf9f2bce35cf95e

**Pull request closed without merging**